intake_esm/core.py

```diff
@@ -163,6 +167,10 @@ def keys_info(self) -> pd.DataFrame:
 
         """
         results = self.esmcat._construct_group_keys(sep=self.sep)
+        if self.esmcat.aggregation_control and self.esmcat.aggregation_control.groupby_attrs:
+            pass
+        else:
+            pass
```

Comment:
is this still necessary?

```suggestion
        if self.esmcat.aggregation_control and self.esmcat.aggregation_control.groupby_attrs:
            pass
        else:
            pass
```

Reply:
I agree with @andersy005 - is this still neccessary?